yoga/Yoga.c

```diff
@@ -179,10 +179,26 @@ static inline float YGComputedEdgeValue(const float edges[YGEdgeCount],
   }
 
   if (edge == YGEdgeStart || edge == YGEdgeEnd) {
-    return YGUndefined;
+    YGValue result;
+    result.value = YGUndefined;
+    result.defined = false;
+    result.unit = YGUnitPixel;
+    return result;
   }
 
-  return defaultValue;
+  YGValue result;
+  result.value = defaultValue;
+  result.defined = (defaultValue < 0 || defaultValue >= 0); /* is faster than a nan function call and enough at this point */
+  result.unit = YGUnitPixel;
+  return result;
```

Comment:
VS15 has support for this syntax and it is used throughout the project so i suggest upgrading. I would prefer it and it is part of C99. Is there a reason you are using a old VS build? if there are legit usages for old VS then I would be OK with leaving as is.